charts/kube-prometheus-stack/values.yaml

```diff
@@ -2853,6 +2853,11 @@ prometheus:
     ##
     retentionSize: ""
 
+    ## Allow out-of-order/out-of-bounds samples ingested into Prometheus for a specified duration
+    ## See https://prometheus.io/docs/prometheus/latest/configuration/configuration/#tsdb
+    # tsdb:
```

Comment:
does this value has a default?
if so i would prefer to use that instead of commenting that out.